### ISSUE TITLE
Email:Html document attached to email missing lang=en attribute/value. 

### ISSUE THREAD

**Issue by JohnHewittCriterion** [nciinc/fs-permit-platform#612]
## Notes
There is no lang on the email attachment .html document.
 
Add lang=en to the <html> document.  (Please check the other pages as well)

URL: https://forest-service-trees-staging.app.cloud.gov/christmas-trees/forests/mthood#season-dates 

![image.png](https://images.zenhubusercontent.com/5913513251aa2c30577de73e/24c7484c-70cc-46f4-8d72-80ef0267ecc6)

Linkage: CTF-95
WCAG: 3.1.1 Language of Page 

## Acceptance criteria
- [ ] email has lang=en (will)

## Tasks
- [ ] add lang=en

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by sadlerw** [comment]
<img width="268" alt="screen shot 2018-03-22 at 2 25 18 pm" src="https://user-images.githubusercontent.com/2058839/37793510-e9c86814-2ddc-11e8-8ab0-9febadf9ef4f.png">

<img width="754" alt="screen shot 2018-03-22 at 2 24 01 pm" src="https://user-images.githubusercontent.com/2058839/37793517-eff785d0-2ddc-11e8-851a-cfdd102f9eae.png">
